system-parachains/asset-hubs/asset-hub-kusama/src/weights/pallet_xcm.rs

```diff
@@ -47,6 +47,9 @@ use core::marker::PhantomData;
 /// Weight functions for `pallet_xcm`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
+	fn claim_assets() -> Weight {
+		todo!()
```

Comment:
Don't integration tests blow up on hitting this?

Reply:
good point, @franciscoaguirre I usually copy missing weights from `polkadot-sdk, anyway they will be regenerated before release.

And also I think nor  xcm emulated tests neither unit-tests triggers weight calculation for fees, so it wont fail.
They only tests that triggers that are our bridge tests where we use this: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/bridge-hubs/bridge-hub-kusama/tests/tests.rs#L91-L98

Reply:
I usually leave it as `todo!()` and then regenerate the weights, but since they'll be all regenerated before release, I'll just input some number.

Reply:
better to use `Weight::MAX` or similar, so that if we ever miss updating one of them we just fail that extrinsic with overweight error rather than crash the runtime 💣

Reply:
Added weights copied from `polkadot-sdk` for now